### PR TITLE
Improve wording in CoC regarding info to include in report.

### PR DESCRIPTION
**Pull request by mxsasha**
Thanks to @ddbeck for the suggestion.

This was the only point of feedback raised in https://trello.com/c/XcsL0bAW/194-feedback-on-code-of-conduct-update

### CODE REVIEW

**Review by ddbeck — APPROVED** [review]
This is great. Thank you!

File: docs/code-of-conduct.rst [hunk 1]
```diff
@@ -117,14 +117,14 @@ If you are not sure whether the situation was a Code of Conduct violation, or wh
...
-* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+* Names or descriptions of anyone who was involved or witness to the incident.
 * When and where the incident occurred. Please be as specific as possible.
 * Your account of what occurred. If there is a written record (e.g. tweets or slack messages) please include screenshots, or otherwise a link.
 * Any extra context you believe existed for the incident.
 * If you believe this incident is ongoing.
 * Any other information you believe we should have.
 
-If you don’t have some of this information, or not at this time, please still make the report anyways.
+If you don’t have some of this information, or not at this time, please still make the report and include as much information as you have.
```
Comment (plaindocs) [inline docs/code-of-conduct.rst]:
```suggestion
If you don’t all of this information at the time, please still make the report and include as much information as you have.
```

File: docs/code-of-conduct.rst [hunk 2]
```diff
@@ -117,14 +117,14 @@ If you are not sure whether the situation was a Code of Conduct violation, or wh
 In your report please include, when possible:
 
 * Your contact info (so we can get in touch with you)
-* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+* Names or descriptions of anyone who was involved or witness to the incident.
```
Comment (plaindocs) [inline docs/code-of-conduct.rst]:
```suggestion
* Names or descriptions of anyone who was involved or who witnessed the incident.
```

**Review by plaindocs — CHANGES_REQUESTED** [review]
Looks good. Couple of suggestions.

**Review by plaindocs — APPROVED** [review]
:shipit:

**Pull request merged**